README.rst

```diff
@@ -104,6 +104,17 @@ This directive is used to document generic package instantiations. It's of the f
 
         :instpkg: GNATCOLL.Opt_Parse.Parse_Option
 
+``generic_package`` directive
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This directive is used to document packages. It's of the form::
```

Comment:
This description doesn’t mention “generic” and the name of the directive below looks wrong